index.d.ts

```diff
@@ -5,6 +5,11 @@ declare namespace waitForLocalhost {
 		 */
 		port?: number;
 
+		/**
+		@default '/''
+		 */
```

Comment:
```suggestion
		*/
```

Reply:
Space character is on the existing options, will remove from all. 🎯

Reply:
✔️